services/tenant-ui/frontend/src/main.ts

```diff
@@ -37,6 +37,22 @@ async function loadApp() {
       after((result) => {
         console.log('configuration loaded from server.');
         console.log(result);
+        // Matomo Setup
```

Comment:
Does this need to be done in the callback handler? Or could it just be done below after the unsubscribe, using the configStore getter to check the value?

It makes a bit more sense to me to leave this action handler to specifically loading the config, and then if using it to set up matomo doing that below with the other setup steps.

Minor though, fine to handle it here if necessary for now.

Reply:
Makes a lot of sense. Not much point in complicating this.